hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/NodeLabelsPage.java

```diff
@@ -33,10 +33,14 @@ public class NodeLabelsPage extends RouterView {
   protected void preHead(Hamlet.HTML<__> html) {
     commonPreHead(html);
     String type = $(NODE_SC);
+    String nodeLabel = $(NODE_LABEL);
     String title = "Node labels of the cluster";
```

Comment:
Should we use a StringBuilder?

Reply:
Thanks for your suggestion! I will use `Stringbuilder` to improve this part of the code.